README.md

```diff
@@ -19,8 +19,17 @@ community, but require consensus from domain specific project maintainers in
 order to implement and accept into the release.
 
 For an overview of the whole project, see [the roadmap](ROADMAP.md).
+
 For a quick-start, FAQ, and template references, see [the guidelines](guidelines/README.md).
 
+## Why are Enhancements Tracked?
+
+As the project evolves, its important that the OKD community understands how we
```

Comment:
Yeah, I decided to leave it as-is for now because we do have some OKD changes in the pipeline and I'd like for those to settle before we update the docs here to include OKD.